.github/workflows/deploy_app.yml

```diff
@@ -14,8 +14,19 @@ env:
  DOCKERHUB_REPOSITORY: dfedigital/teaching-vacancies
 
 jobs:
+  turnstyle:
+    runs-on: ubuntu-20.04
+    timeout-minutes: 20
+    steps:
+    - uses: softprops/turnstyle@v1
+      name: Check workflow concurrency
+      with:
+        poll-interval-seconds: 20
+        same-branch-only: true
+      env:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
   deploy-app:
-    name: Deploy app to environment
+    name: Deploy ${{tag}} to ${{environment}}
```

Comment:
Are `tag` and `environment` defined somewhere? And should we put spaces on the inside of the brackets? (we tend to elsewhere)

Reply:
`tag` and `environment` are defined as inputs to this workflow in lines 3-11:
```
on:
  workflow_dispatch:
    inputs:
      environment:
        description: Environment
        required: true
      tag:
        description: Docker tag
        required: true
```

In looking at the spacing inside the brackets, I've made the values more explicit:
```
name: Deploy ${{ github.event.inputs.tag }} to ${{ github.event.inputs.environment }}
```